### PR TITLE
issue-2983: Update doc titles

### LINKED ISSUE #2983: Shortened guide titles need link updates in code for Sat
The guide titles shortened in https://github.com/theforeman/foreman-documentation/pull/2874 require link updates in code in [documentation.rb in foreman_theme_satellite](https://github.com/RedHatSatellite/foreman_theme_satellite/blob/develop/lib/foreman_theme_satellite/documentation.rb)
Cherry-pick to 6.15 needed.

Btw, before you do that, I suggest "Managing configurations using X integration" is changed to "Managing configurations **by** using X integration" for translation reasons.

### PR DESCRIPTION
**Pull request by apinnick**
Fixes [2983](https://github.com/theforeman/foreman-documentation/issues/2983)

For 6.16.

### CODE REVIEW

**Comment by evgeni** [comment]
This will require an update to the toc file.
CC @ShimShtein

**Comment by ShimShtein** [comment]
@evgeni : https://github.com/RedHatSatellite/foreman_theme_satellite/pull/47

File: test/fixtures/toc.json [hunk 1]
```diff
@@ -635,6 +674,9 @@
     ]
   },
   "managing_configurations_using_ansible_integration_in_red_hat_satellite": {
```
Comment (evgeni) [inline test/fixtures/toc.json]: according to the changes to `documentation.rb`, this was renamed to `managing_configurations_using_ansible_integration`, while it is not renamed here?

Comment (ShimShtein) [reply test/fixtures/toc.json]: I had to update the book-to-path mapping file. Now it's OK and the failures are relevant.

**Comment by ShimShtein** [comment]
@apinnick , the fails are actually relevant: https://github.com/RedHatSatellite/foreman_theme_satellite/actions/runs/8897047565/job/24431151567?pr=46#step:21:12

**Comment by apinnick** [comment]
> @apinnick , the fails are actually relevant: https://github.com/RedHatSatellite/foreman_theme_satellite/actions/runs/8897047565/job/24431151567?pr=46#step:21:12

@ShimShtein That failure is strange. When I checked the failed links manually, they all worked. I am not sure how the "TOC" works, however.

**Comment by ShimShtein** [comment]
The TOC file is actually the extract of the documentation links from not-yet-published docs downstream. This is the link structure that will be available later on. So we can anticipate any changes to the doc structure before the docs are actually published.

**Review by Lennonka — COMMENTED** [review]
Needs cherrypick to 6.15!

**Comment by ShimShtein** [comment]
@evgeni I don't see a stable branch for the 6.15 equivalent version. What is our policy for this repo?

**Comment by evgeni** [comment]
We don't have any breaking changes since the version that is in 6.15, so I think we can just push whichever the next release will be?

**Comment by ShimShtein** [comment]
Apparently the documentation structure is about to change (see the failures in this PR).
We will need a version with documentation links as they are in 6.15 (with the fixes from this PR) and one for the next documentation structure.

**Comment by evgeni** [comment]
Well, then we'll create a stable branch the moment we're breaking it :)

**Comment by ShimShtein** [comment]
I think it's now - the change as it is in the original PR needs to be done to 6.15, and the new TOC + changes that come with it should go to the next version.

**Comment by evgeni** [comment]
The TOC in this PR, is it for 6.15 or for "develop"?
The documentantion.rb in this PR, is it for 6.15 or "develop"?

**Comment by ShimShtein** [comment]
The TOC is for `develop`, the documentation.rb is for `6.15` (but it will need adjustments for `develop` too - the errors that we see in the PR right now)

**Comment by evgeni** [comment]
ok, can we have a toc for 6.15, merge the (test passing) result, then update both to develop and then we  have a clean branch point

**Comment by ShimShtein** [comment]
Updated the TOC to 6.15

**Comment by ShimShtein** [comment]
Finally all green

**Comment by apinnick** [comment]
> Finally all green

@ShimShtein Thanks!

I tried to figure out the problem on my own but didn't succeed. :-(

**Review by evgeni — APPROVED** [review]

**Pull request merged**